README.adoc

```diff
@@ -414,6 +415,7 @@ This container includes the model, so you can use it to perform inference at the
 11. *Deploy a web application that uses the deployed model via the Model Service REST API*.
 
 a. From the developer perspective of the OpenShift web console, and click btn:[+Add], then click btn:[Import YAML].
+// TechEditor:[style] This sentence does not make sense. I think the "and" is misplaced. If you remove the "and" then the sentence makes sense. I wasnt sure if you had more intention to introduce something here so I have lf it up to you to take a look at. 
```

Comment:
Yes, I think I rewrote the sentence at some point and forgot the remove the "and".

```suggestion
a. From the developer perspective of the OpenShift web console, click btn:[+Add], then click btn:[Import YAML].
// TechEditor:[style] This sentence does not make sense. I think the "and" is misplaced. If you remove the "and" then the sentence makes sense. I wasnt sure if you had more intention to introduce something here so I have lf it up to you to take a look at. 
```